src/Symfony/Component/Config/Resource/ClassExistenceResource.php

```diff
@@ -121,20 +128,38 @@ public function unserialize($serialized)
      *
      * @internal
      */
-    public static function throwOnRequiredClass($class)
+    public static function _throwOnRequiredClass($class)
     {
         if (self::$autoloadedClass === $class) {
             return;
         }
+
+        self::throwOnRequiredClass($class, null);
+    }
+
+    /**
+     * @throws \ReflectionException When $class is not found and is required
+     *
+     * @internal
+     */
+    public static function throwOnRequiredClass($class, \Exception $previous = null)
```

Comment:
All `\Exception` should be made `\Throwable` when merging to upper branches that supports PHP 7.

Reply:
not the case here: we don't want to swallow Error - only Exception